llvm/include/llvm/CodeGen/ISDOpcodes.h

```diff
@@ -1451,6 +1451,11 @@ enum NodeType {
   VECREDUCE_UMAX,
   VECREDUCE_UMIN,
 
+  // The `llvm.experimental.vector.partial.reduce.add` intrinsic
+  // Operands: Accumulator, Input
+  // Outputs: Output
+  PARTIAL_REDUCE_ADD,
```

Comment:
Should this be `EXPERIMENTAL_PARTIAL_REDUCE_ADD` (like `EXPERIMENTAL_VECTOR_HISTOGRAM`)?